gtest-parallel

```diff
@@ -420,30 +419,32 @@ tests.sort(reverse=True, key=lambda x: ((1 if x[0] is None else 0), x))
 tests = [(test, i + 1) for test in tests for i in range(options.repeat)]
 test_lock = threading.Lock()
 job_id = 0
-logger.log(str(-1) + ': TESTCNT ' + ' ' + str(len(tests)))
+logger.log(-1, "TESTCNT", (len(tests),))
 
 exit_code = 0
 
+# Remove files from old test runs.
+if os.path.isdir(options.output_dir):
+  shutil.rmtree(options.output_dir)
 # Create directory for test log output.
 try:
-  os.makedirs(options.output_dir)
+  for result in 'passed', 'failed', 'interrupted':
```

Comment:
I think these should only be made if there are any failed/interrupted tests, so you can do this in the code that moves the logs.